kyuubi-server/src/test/scala/org/apache/kyuubi/events/EventLoggingServiceSuite.scala

```diff
@@ -21,13 +21,17 @@ import java.net.InetAddress
 import java.nio.file.Paths
 import java.util.UUID
 
+import com.fasterxml.jackson.databind.ObjectMapper
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.kyuubi
```

Comment:
nit: move down one line

Reply:
It's the `scalastyle-config.xml` file problem. Maybe we need to improve it.
![Screenshot from 2021-11-23 15-48-23](https://user-images.githubusercontent.com/29809822/142987242-6f801dc3-c6fe-42ff-92b1-f09c9dc47314.png)
![Screenshot from 2021-11-23 15-48-41](https://user-images.githubusercontent.com/29809822/142987250-e16b236f-0d48-4c1e-b72b-19f7bf39d097.png)

Reply:
if so, we can do this in followup. aslo cc @pan3793 @zhouyifan279

Reply:
Or we can just import all without using the qualifier `kyuubi`.

Reply:
does `dev/reformat` work?

Reply:
IDEA => Preferences => Editor => Code Style => Scala => Change Schema to Project, change Formatter to scalafmt, then you can use ⌥ ⌘ L to format code & reorder imports. Notes, ⌃ ⌥ O doesn't respect `scalafmt.conf`, which will break the import order

Reply:
I think it's not idea problem.
![Screenshot from 2021-11-23 17-09-28](https://user-images.githubusercontent.com/29809822/142997209-5d0e4ed8-63da-4aae-8856-c2a91b4a8482.png)

Reply:
I checked out your branch, it's out of sync master too long, please rebase master

Reply:
I can not reproduce the issue.
The whole step: (omit part of the output)
```
➜  apache-kyuubi git:(master) dev/checkout_pr.sh -f 1378
➜  apache-kyuubi git:(PR_1378) dev/reformat
➜  apache-kyuubi git:(PR_1378) build/mvn clean install -DskipTests
[INFO] Reactor Summary for Kyuubi Project Parent 1.4.0-SNAPSHOT:
[INFO]
[INFO] Kyuubi Project Parent .............................. SUCCESS [  5.320 s]
[INFO] Kyuubi Project Common .............................. SUCCESS [01:02 min]
[INFO] Kyuubi Project Embedded Zookeeper .................. SUCCESS [  9.731 s]
[INFO] Kyuubi Project High Availability ................... SUCCESS [ 14.090 s]
[INFO] Kyuubi Project Control ............................. SUCCESS [ 15.654 s]
[INFO] Kyuubi Project Metrics ............................. SUCCESS [ 14.713 s]
[INFO] Kyuubi Project Download Externals .................. SUCCESS [  6.710 s]
[INFO] Kyuubi Project Hive JDBC Client .................... SUCCESS [  5.145 s]
[INFO] Kyuubi Project Hive JDBC Shaded Client ............. SUCCESS [ 20.991 s]
[INFO] Kyuubi Project Engine Spark SQL .................... SUCCESS [ 32.978 s]
[INFO] Kyuubi Project Server .............................. SUCCESS [ 40.353 s]
[INFO] Kyuubi Project Dev Code Coverage ................... SUCCESS [  2.107 s]
[INFO] Kyuubi Project Assembly ............................ SUCCESS [  1.050 s]
[INFO] Kyuubi Project Hive Beeline ........................ SUCCESS [  1.482 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:52 min
[INFO] Finished at: 2021-11-24T10:05:15+08:00
[INFO] ------------------------------------------------------------------------
```
@zhouyifan279 would you please test it on Ubuntu 20.04?

Reply:
I executed above commands on a newly cloned local repository, the result seems fine.

![image](https://user-images.githubusercontent.com/88070094/143160028-3f7e7f9a-a8be-47c0-b0ec-abe938c2925c.png)

Reply:
> I executed above commands on a newly cloned local repository, the result seems fine.
> 
> ![image](https://user-images.githubusercontent.com/88070094/143160028-3f7e7f9a-a8be-47c0-b0ec-abe938c2925c.png)

Yes, it's OK. I mean, when we move down `import org.apahce.kyuubi` one line, it's not ok. Because style check error.
I will use import `import org.apahce.kyuubi._` or other way to resolve it later.

Reply:
@zhouyifan279 I think it's not fine ..  your snapshot shows the package `org.apache.kyuubi` is belong to third-party but we should mark it as kyuubi self.

Reply:
I digged into scalafmt's source code and found that scalafmt always uses the longest prefix to match import groups, eg, use prefix `org.apache.` of `org,apache.kyuubi`.

I'm afraid we have to always use `org.apache.kyuubi._` instead of `org.apache.kyuubi` to accommodate scalafmt.

Reply:
I created a issue to track it, https://github.com/apache/incubator-kyuubi/issues/1452